osf/migrations/0002_auto_20170329_1729.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2017-03-29 22:29
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import osf.utils.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='osfuser',
+            name='mailing_lists',
+        ),
+        migrations.AlterField(
+            model_name='abstractnode',
+            name='date_created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='abstractnode',
+            name='date_modified',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now=True, db_index=True, default=django.utils.timezone.now),
```

Comment:
It's because it was previously nullable. Django uses this to populate empty cells when migrating:

```
You are trying to change the nullable field 'date_modified' on abstractnode to non-nullable without a default; we can't do that (the database needs something to populate existing rows).
Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows)
 2) Ignore for now, and let me handle existing rows with NULL myself (e.g. because you added a RunPython or RunSQL operation to handle NULL values in a previous data migration)
 3) Quit, and let me add a default in models.py
Select an option: 1
Please enter the default value now, as valid Python
The datetime and django.utils.timezone modules are available, so you can do e.g. timezone.now()
>>> timezone.now
```